### DISCUSSION TITLE
Add setting to allow comment based on cursor or highlight

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4025] [Ideas]
Actually I will just mention this one in the Ideas section for future developers.

Allow when cursor is in a line for comment toggle to use a single line comment symbols, and when multiple lines are hightlight and user uses comment toggle that it uses multi line comment symbols, both based on the current documents coding language used.

Thank You

**Comment by elextr** [comment]
As explained on #3997 it is risky that the delimiting comments nest safely.  Probably not a good idea.
